srml/fees/src/lib.rs

```diff
@@ -14,7 +14,114 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Handles all transaction fee related operations
+//! ## Overview
+//! 
+//! The `fees` module handles transaction fee related operations. To use this module, the [`TransferAsset`](https://crates.parity.io/srml_fees/trait.Trait.html#associatedtype.TransferAsset) trait must be implemented to enable asset transfer between accounts. This module provides the following functionality
```

Comment:
it seems to me `use` can be understand in two way here: a module using this module (in case the sentence doesn't really make sense as contract doesn't implement the transferAsset trait) or a runtime (which seems to be the meaning here but is also the second meaning that came to my mind as this module is really supposed to be an user end module as dispatchable are used only by other module)
As the first meaning came to my mind I was confused, I would write something like `this module requires a module that implement TransferAsset...`